sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -46,8 +46,8 @@ mkdir $SYCL_HOME
 
```

Comment:
Please, specify that these instructions are valid only for 64-bit Windows/MSVC installation.
E.g. `**Windows**` -> `**Windows (64-bit)**`

Reply:
Please, add a note that 32-bit configuration is not supported.

Reply:
I just noticed this comment. Do you suggest fixing it here only or everywhere below?

Reply:
Ok, Fixed.